PictureApp/Model/CheckTableViewCell.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  CheckTableViewCell.swift
+//  PictureApp
+//
+//  Created by Irina Gorbachenko on 08.12.2021.
+//
+
+import UIKit
+
+protocol CheckTableViewCellDelegate: AnyObject {
+
+  func checkTableViewCell(_ cell: CheckTableViewCell, didChagneCheckedState checked: Bool)
+}
+
+class CheckTableViewCell: UITableViewCell {
+    
+    
+     @IBOutlet weak var checkbox: Checkbox!
+     @IBOutlet weak var label: UILabel!
+     
+     weak var delegate: CheckTableViewCellDelegate?
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
```

Comment:
I think you could remove this init, because you use init coder

Reply:
Fixed